en/dev_log/ulog_file_format.md

```diff
@@ -41,233 +48,318 @@ The file consists of three sections:
 ----------------------
 ```
 
+A description of each section is provided below.
+
+
 ### Header Section
 
 The header is a fixed-size section and has the following format (16 bytes):
+
 ```
 ----------------------------------------------------------------------
 | 0x55 0x4c 0x6f 0x67 0x01 0x12 0x35 | 0x01         | uint64_t       |
 | File magic (7B)                    | Version (1B) | Timestamp (8B) |
 ----------------------------------------------------------------------
 ```
 
-Version is the file format version, currently 1.
-Timestamp is a `uint64_t` integer, denotes the start of the logging in microseconds.
+- **File Magic (7 Bytes):** File type indicator that reads "ULog"
```

Comment:
Note the two characters at the end. THese don't render in gitbook, and Notepad shows them as: 
![image](https://user-images.githubusercontent.com/5368500/169185672-cddf9e1f-0846-4485-a24c-36669035aa1d.png)

Is there another way to describe this - e.g. "..that reads "ULogXY, where XY is [whatever it is]"

Reply:
> e.g. "..that reads "ULogXY, where XY is [whatever it is]"

I can say "ULogXYZ, where XYZ are the magic 3 bytes: 0x01 0x12 0x35"

Reply:
That works.